src/probability/kernel/measurable_integral.lean

```diff
@@ -0,0 +1,161 @@
+/-
+Copyright (c) 2023 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import probability.kernel.basic
+
+/-!
+# Measurability of the integral against a kernel
+
+The Lebesgue integral of a measurable function against a kernel is measurable.
+
+## Main statements
+
+* `measurable_lintegral`: the function `a ↦ ∫⁻ b, f a b ∂(κ a)` is measurable, for an s-finite
```

Comment:
```suggestion
* `probability_theory.kernel.measurable_lintegral`: the function `a ↦ ∫⁻ b, f a b ∂(κ a)` is
  measurable, for an s-finite
```
for the sake of docgen

Reply:
Thanks! Making the names docgen-compatible is indeed something I tend to forget.